.github/workflows/build.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+#
+# Build checked out freertos-pkcs11-psa submodule against amazon-freertos tree.
+#
+# Copyright (c) 2020 Linaro Limited
+#
+# Based on instructions:
+# https://docs.aws.amazon.com/freertos/latest/userguide/getting_started_cypress_psoc64.html
+#
+# This script assumes:
+# 1. A particular revision of the freertos-pkcs11-psa repo was cloned under
+# "freertos-pkcs11-psa" subdirectory.
+# 2. The gnuarmemb toolchain is already in the PATH.
+#
+# This script is supposed to be usable both as part of Github Workflow and
+# for local testing/debugging, please keep it such.
```

Comment:
If for the local test usage, the build.sh script should only be run in the path above the freertos-pkcs11-psa, right? If so, how about adding some usage guide or note for this script?

Reply:
Thanks for reviewing. Well, fairly speaking, intention to keep it locally-usable was a kind of background feature, based on experience with other CI setups, that debugging issues in the real CI may be too time-consuming/cumbersome, and it's helpful to be able to run it locally easily enough.

I now see I called for extra attention to that with that "please keep it such" phrase, which more along the lines of "don't rely on Github Actions environment too much, keep it runnable on other systems too".

Well, I'll look how long instructions for local running would be, and add them to code comments or commit message (if too long to be inline).

Reply:
@Sherryzhang2, so, I updated a little bit (but just a little bit) a section above in the file. It now reads:

~~~
# This script assumes:
# 1. The gnuarmemb toolchain is already in the PATH.
# 2. A particular revision of the freertos-pkcs11-psa repo was cloned under
# "freertos-pkcs11-psa" subdirectory of the current directory.
~~~

And well, those are pretty much *the* instructions on how to run it locally. Again, local running isn't supposed to be a core feature, it just should be possible at all, for (hopefully rare) debugging purposes. So, I don't think it makes sense to give overdetailed instructions regarding that. If in some time you, I, or somebody else will find them confusing or some points missing, we can always extend them. Of course, if you have specific suggestions now, please share them.